src/elements/kw-share-card/kw-share-card.html

```diff
@@ -135,7 +135,7 @@
                             <p>[[share.description]]</p>
                         </div>
                         <div class="actions">
-                            <button class="orange-button remix-button">remix<img src="/assets/icons/shares/remix.svg" /></button>
+                            <button on-tap="_onTapRemix" class="orange-button remix-button">remix<img src="/assets/icons/shares/remix.svg" /></button>
```

Comment:
Remix action should be handled by the card itself, check the current behaviour of kano world